### PR TITLE
Feature/textarea component

### PR DESCRIPTION
**Pull request by 0xRowdy**
## GitHub Issue

[GitHub issue](https://github.com/HausDAO/daohaus-monorepo/issues/21#issue-1190114529)

## Changes

Added atom for TextArea
Added molecule for WrappedTextArea
Added disabled placeholder text style

## Packages Added

Brief list of any packages added, and if folks need to rerun `yarn install` to run locally
None

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: libs/ui/src/components/atoms/TextArea/TextArea.styles.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+
+import { Theme } from '../../../types/theming';
+import { field } from '../../../theme/component/fieldFamily';
+import { font } from '../../../theme/global/font';
```
Comment (jonathanprozzi) [inline libs/ui/src/components/atoms/TextArea/TextArea.styles.ts:5]: Doesn't seem like we use this import -- can remove if unneeded

File: libs/ui/src/components/atoms/TextArea/TextArea.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { IconType } from 'react-icons';
```
Comment (jonathanprozzi) [inline libs/ui/src/components/atoms/TextArea/TextArea.tsx:2]: Seems unused as well

**Review by jonathanprozzi — APPROVED** [review]
Nice job! I'm really liking this structure of `<Atom/>` and `<WrappedAtom/>` for these components so far

**Comment by 0xRowdy** [comment]
> Nice job! I'm really liking this structure of `<Atom/>` and `<WrappedAtom/>` for these components so far

Yeah starting to take shape and a pattern form.

**Pull request merged**